### ISSUE TITLE
update to https

### ISSUE THREAD

**Issue by michael-lynch** [jmnunezizu/passport-discogs#1]
Discogs now only supports `https`. Without it, the callback URL won't work properly.

```
options.requestTokenURL = options.requestTokenURL || 'https://api.discogs.com/oauth/request_token';
options.accessTokenURL = options.accessTokenURL || 'https://api.discogs.com/oauth/access_token';
options.userAuthorizationURL = options.userAuthorizationURL || 'https://www.discogs.com/oauth/authorize';
```

**Comment by jmnunezizu** [comment]
Hi @michael-lynch,

Oh brilliant, thanks for the heads up.

I'll try to incorporate the change and cut a release throughout this week. If you happen to have it already, please feel free to raise a PR.

Cheers,
Jose.-

**Comment by jmnunezizu** [comment]
Hi @michael-lynch,

I made the change, and also changed the `identity` endpoint to use HTTPS.

I've released this with version `0.2.0` and published it to npm. Give it a go.

Cheers,
Jose.-
